lib/attributor/types/csv.rb

```diff
@@ -29,5 +29,9 @@ def self.example(context=nil, options: {})
       return collection.join(',')
     end
 
+    def self.family
```

Comment:
I guess...by the time an example is rendered, it will be a string...yes, but if you get the object in memory, so to speak you might want to "represent" it differently, or know that it really is an array of things...hmmm.